data/iterations.json

```diff
@@ -0,0 +1,134 @@
+[
+    {
+        "from": "2018-12-03",
+        "to": "2018-12-03"
+    },
+    {
+        "from": "2019-01-23",
+        "to": "2019-01-23"
+    },
+    {
+        "from": "2019-02-18",
+        "to": "2019-02-18"
+    },
+    {
+        "from": "2019-03-04",
+        "to": "2019-03-04"
+    },
+    {
+        "from": "2019-03-18",
+        "to": "2019-03-18"
+    },
+    {
+        "from": "2019-04-02",
+        "to": "2019-04-02"
+    },
+    {
+        "from": "2019-04-15",
+        "to": "2019-04-15"
+    },
+    {
+        "from": "2019-04-29",
+        "to": "2019-04-29"
+    },
+    {
+        "from": "2019-05-13",
+        "to": "2019-05-13"
+    },
+    {
+        "from": "2019-06-04",
+        "to": "2019-06-04"
+    },
+    {
+        "from": "2019-06-24",
+        "to": "2019-06-24"
+    },
+    {
+        "from": "2019-06-24",
+        "to": "2019-06-24"
+    },
+    {
+        "from": "2019-08-19",
+        "to": "2019-08-19"
+    },
+    {
+        "from": "2019-09-09",
+        "to": "2019-09-09"
+    },
+    {
+        "from": "2019-09-16",
+        "to": "2019-09-16"
+    },
+    {
+        "from": "2019-09-30",
+        "to": "2019-09-30"
+    },
+    {
+        "from": "2019-10-14",
+        "to": "2019-10-14"
+    },
+    {
+        "from": "2019-11-04",
+        "to": "2019-11-04"
+    },
+    {
+        "from": "2019-11-18",
+        "to": "2019-11-18"
+    },
+    {
+        "from": "2019-12-07",
+        "to": "2019-12-07"
+    },
+    {
+        "from": "2020-01-01",
+        "to": "2020-01-01"
+    },
+    {
+        "from": "2020-02-01",
+        "to": "2020-02-01"
+    },
+    {
+        "from": "2020-02-15",
+        "to": "2020-02-15"
+    },
+    {
+        "from": "2020-03-03",
+        "to": "2020-03-03"
+    },
+    {
+        "from": "2020-03-03",
+        "to": "2020-03-03"
+    },
+    {
+        "from": "2020-03-30",
+        "to": "2020-03-30"
+    },
+    {
+        "from": "2020-04-11",
+        "to": "2020-04-11"
+    },
+    {
+        "from": "2020-04-20",
+        "to": "2020-04-20"
+    },
+    {
+        "from": "2020-05-04",
+        "to": "2020-05-04"
+    },
+    {
+        "from": "2020-05-16",
+        "to": "2020-05-16"
+    },
+    {
+        "from": "2020-05-30",
+        "to": "2020-05-30"
+    },
+    {
+        "from": "2020-06-15",
+        "to": "2020-06-26"
+    },
+    {
+        "from": "2020-06-29",
+        "to": "2020-07-10"
+    }
+]
```

Comment:
releases.json and issues.csv are independent from all other files; they update every time they're run. iterations.json cuts up issues.csv to create iteration-summary.json.

Reply:
can we add a date generated to these files so we know the last time they were updated?

Reply:
I'm not seeing `velocity_iteration.py` anywhere, trying to find it by ctrl+f and this is the only mention of velocity on the page.

Reply:
Sorry, I renamed the file in a later commit. Here it is corrected:

> `releases.json` is built by `releases.py`. On each run, new rows are appended to `issues.csv` by `get_issues.py`. `summarize_iterations.py` uses `iterations.json` to cut up `issues.csv` to create `iteration-summary.json`.